### PR TITLE
tentative fix for artifactory

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

**Comment by wind57** [comment]
In a weird twist of events, we have moved to artifactory. And `helm deploy` started failing. The fix for this is trivial. The header  `X-JFrog-Art-Api` is documented in the official documentation to be needed. Take a look and let me know what you think. thank you

File: src/main/java/com/kiwigrid/k8s/helm/HelmRepository.java [hunk 1]
```diff
@@ -17,6 +18,9 @@
 
 	private String password;
 
+	// X-JFrog-Art-Api from artifactory
+	private String apiKey;
```
Comment (wind57) [inline src/main/java/com/kiwigrid/k8s/helm/HelmRepository.java:22]: if you agree with this change, may be we could rename this variable to something like `artfactoryApiKey`, also I most probably should update the documentation also

**Comment by jekkel** [comment]
If I read [the documentation](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API) correctly:

> Artifactory's REST API supports these forms of authentication:
>
>  * Basic authentication using your username and password
>  * Basic authentication using your username and API Key.
>  * Using a dedicated header (X-JFrog-Art-Api) with your API Key.

Upload with API Key authentication should also work via Basic Auth hence with the existing implementation. Did you try that?

**Comment by wind57** [comment]
you' right. (face-palm). This was indeed a problem at our end. :( closing.

**Pull request closed without merging**